src/infrastructure/validation/logger/LoggerManager.java

```diff
@@ -67,7 +80,51 @@ protected LoggerManager() {
 			consoleLogger = new ConsoleLogger(enabledLogLevelsList);
 		}
 	}
+	
+	/** 
+	 *  helper method that checks whether the logger is to be initialized in test mode,
+	 *  works by the testMode tag in the default XML file
+	 *  If no such tag exists, then the file is considered to be not in test mode and so,
```

Comment:
Instead of deleting and rewriting tags in XML to turn on or off the testmode. Why not have a test mode boolean in the xml? Then we don't need to keep writing the path everytime we want test mode, we just change the boolean to true

Reply:
alright. makes sense